### DISCUSSION TITLE
What's the easiest Debian ISO to start from?

### DISCUSSION THREAD

**Discussion by jc00ke** [regolith-linux/regolith-desktop#933] [Q&A]
I tried the net install and even after installing the Regolith packages I had to manually install `lightdm` and `xrescat` and other tools before I could log in to a Regolith on Wayland session. And I couldn't install `regolith-look-solarized-dark` because it relies on `solarc-theme` which couldn't be satisfied (sorry, not at the office anymore)

It was pretty painful to get to there from the net install, so wondering if there's a better base from which to start?

**Comment by Diogenes08** [comment]
I second this question/issue, as I have had the same happen with both Debian 12 and Antix 23, both of which had an identical set of issues.

Aside from lightdm and xrescat, what else if anything did you need to install to make it work? I am still trudging through that same mess, currently, though I am currently only looking to make the X11 version work; while I intend to try Wayland eventually, I am...taking this one step at a time.

**Reply by jc00ke** [reply]
Things I've had to install manually:

- lightdm
- lsb-release
- xrescat
- libdbus-glib-1-dev (for firefox)
- curl
- wget

Things I can
